commit a6439d797ebb257334ae60f5f986d1533cba599d
Author: Tamir Duberstein <tamird@gmail.com>
Date:   Mon Apr 17 23:52:27 2017 -0700

    CMake: compile with -O2
    
    Summary:
    Seems pretty important.
    Closes https://github.com/facebook/rocksdb/pull/2161
    
    Differential Revision: D4891097
    
    Pulled By: yiwu-arbug
    
    fbshipit-source-id: 16dc0149dc89d426407fabab4a5ba1d6d1562269

diff --git a/CMakeLists.txt b/CMakeLists.txt
index cdf9feef..1290d4a6 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -138,7 +138,7 @@ else()
   endif()
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
   if(NOT CMAKE_BUILD_TYPE STREQUAL "Debug")
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fno-omit-frame-pointer")
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -O2 -fno-omit-frame-pointer")
     include(CheckCXXCompilerFlag)
     CHECK_CXX_COMPILER_FLAG("-momit-leaf-frame-pointer" HAVE_OMIT_LEAF_FRAME_POINTER)
     if(HAVE_OMIT_LEAF_FRAME_POINTER)
